pkg/template/ytt.go

```diff
@@ -139,6 +139,11 @@ func (t *Ytt) addInlinePaths(args []string) ([]string, *memdir.TmpDir, error) {
 
 	inline := t.fetchFactory.NewInline(*t.opts.Inline, t.genericOpts.Namespace)
 
+	// TODO / question:should we be creating the path at this point rather than doing the o.stat and conditional create one layer deeper?
+	// TODO / question: does this suffice to get the path cleaned up later? is this important? do we need to do something else to get the path cleaned up later?
```

Comment:
it's not clear to me whether it matters that we append the path to t.opts.Paths --  on the pod it seems like `/etc/kappctrl-mem-tmp/` gets wiped entirely.